pkg/relay/relay.go

```diff
@@ -287,6 +296,15 @@ func (rs *Relay) GetPayload(ctx context.Context, payloadRequest *types.SignedBli
 
 	// defer put delivered datastore write
 	go func(rs *Relay, slot structs.Slot, trace structs.DeliveredTrace) {
+		if rs.config.PublishBlock {
+			beaconBlock := structs.SignedBlindedBeaconBlockToBeaconBlock(payloadRequest, payload.Payload.Data)
+			if err := rs.beacon.PublishBlock(beaconBlock); err != nil {
+				logger.WithError(err).Warn("fail to publish block to beacon node")
+			} else {
+				logger.Info("published block to beacon node")
```

Comment:
also `rs.l.` :)

Reply:
Why? `logger` is initialized in the beginning of the method, and it contains fields for describing the request

Reply:
👍